### PR TITLE
Update Valine & add post count  …

### PR DESCRIPTION
**Pull request by Regulust**
## Update Valine & add post count 
1. Update valine.ejs to the latest stable version
2. Add items of Valine: visitors(show in article-meta area), recordIP(optional): 
- enable at comment - valine of _config.yml
3. Update some translations(will be used in post count)
4. Add word count function at article-meta area (like hexo-theme-next):word count and estimated reading time:
- enable at plugins - counter of _config.yml
5. Add some fontawsome icon for article meta

PS:
- count function needs hexo-wordcount: https://github.com/willin/hexo-wordcount
- templete: http://regulus.cc

### CODE REVIEW

File: layout/common/article.ejs [hunk 1]
```diff
@@ -8,8 +8,10 @@
         <% if (post.layout != 'page') { %>
             <div class="article-meta">
                 <%- partial('post/date', { class_name: 'article-date', date_format: null }) %>
-		<%- partial('post/author') %>
+		        <%- partial('post/author') %>
```
Comment (marcofranssen) [inline layout/common/article.ejs]: tabs vs spaces... replace the tab with the correct amount of spaces.

File: layout/common/post/valinecounter.ejs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<% if (theme.comment.valine.visitor) { %>
+   <span id="<%- url_for(page.path) %>" class="leancloud-visitors" data-flag-title="<%= page.title %>">
+      <i class="fa fa-eye"></i>
+      <span class="post-meta-item-text"><%= __('counter.views') + (':')%> </span>
+      <i class="leancloud-visitors-count">1000000</i>
+      </span>
```
Comment (marcofranssen) [inline layout/common/post/valinecounter.ejs]: reduce indent on this line.

File: layout/common/post/date.ejs [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <% if (post.date) { %>
     <div class="<%= class_name %>">
+         <i class="fa fa-calendar"></i>
```
Comment (marcofranssen) [inline layout/common/post/date.ejs]: one space to much

File: layout/common/post/counter.ejs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<% if (theme.plugins.counter) { %>
+    <div class="article-counter">
+         <i class="fa fa-file-word-o"></i>
+         <span class="post-count-item-text"><%=  __('counter.count') + (':')%> </span>
```
Comment (marcofranssen) [inline layout/common/post/counter.ejs]: one space to much indentation.

File: layout/comment/valine.ejs [hunk 5]
```diff
@@ -1,19 +1,26 @@
...
+                    <script src="//unpkg.com/valine"></script>
+                    <script>
+                        var GUEST = ['nick','mail','link'];
+                        var meta = '<%= theme.comment.valine.meta %>';
+                        meta = meta.split(',').filter(function (item) {
+                            return GUEST.indexOf(item)>-1;
+                        });
+                        var notify = '<%= theme.comment.valine.notify %>' == true ? true : false;
+                        var verify = '<%= theme.comment.valine.verify %>' == true ? true : false;
+                        var avatarcdn = '<%= theme.comment.valine.avatar_cdn %>' == true ? true : false;
```
Comment (marcofranssen) [inline layout/comment/valine.ejs]: van be written as 

```
var notify = '<%= theme.comment.valine.notify %>' == true;
```

The thing already evalueates as true or false so no need to make this explicit by setting to true if evaluates to true.

**Comment by Regulust** [comment]
Some minor fix and optimization in commits on Oct 9, 2019. 
THX for @marcofranssen review.

File: layout/common/article.ejs (hunk 1 above)
Comment (Regulust) [reply layout/common/article.ejs]: OK, already replace all tab with spaces in commits on Oct 9, 2019.

File: layout/common/post/valinecounter.ejs (hunk 2 above)
Comment (Regulust) [reply layout/common/post/valinecounter.ejs]: Done in commits on Oct 9, 2019.

File: layout/common/post/date.ejs (hunk 3 above)
Comment (Regulust) [reply layout/common/post/date.ejs]: Done in the latest commit.

File: layout/common/post/counter.ejs (hunk 4 above)
Comment (Regulust) [reply layout/common/post/counter.ejs]: Done in the latest commit.

File: layout/comment/valine.ejs (hunk 5 above)
Comment (Regulust) [reply layout/comment/valine.ejs]: You said right. Already delete and optimize the code in latest commit minor fix and minor fix2.

**Pull request merged**